### ISSUE TITLE
9.27.4 linux-arm64 zip executable won't run, Error: Cannot find module '../../package.json'

### ISSUE THREAD

**Issue by luckyjay** [zwave-js/zwave-js-ui#4027]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

PKG executable

### Z-Wave JS UI version

9.27.4

### ZwaveJS version

Version that came with release PKG

### Describe the bug

Ubuntu 24 LTS, ARM64 (aarch64) on an RPI4, getting a startup failure using the zwave-js-ui-v9.27.4-linux-arm64.zip package, shown below. 9.27.3 is okay.

![image](https://github.com/user-attachments/assets/c0d29fdb-2c92-4a97-ab6f-50890aea2011)

### To Reproduce

Download the arm64 zip, attempt to run the zwave-js-ui executable.

### Expected behavior

Normal execution

**Comment by wallenium** [comment]
same issue here on Proxmox

**Comment by kpanchen** [comment]
Yes, same issue here, 9.27.4 on Ubuntu 24 LTS, 9.27.3 works fine.

**Comment by RWeinberg3** [comment]
Same issue on Windows 11

**Comment by robertsLando** [comment]
@AlCalzone For some reason esbuild is not including @zwave-js/server package.json in bundle starting from final version:

```js
// node_modules/@zwave-js/server/dist-esm/lib/const.js
init_esbuild_import_meta_url_shim();
var import_node_module2 = require("node:module");
var require4 = (0, import_node_module2.createRequire)(__import_meta_url);
var version = require4("../../package.json").version;
var minSchemaVersion = 0;
var maxSchemaVersion = 40;
var applicationName = "zwave-js-server";
var dnssdServiceType = applicationName;
```

Fixing this now. cc @AlCalzone
